apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Optional.kt

```diff
@@ -18,6 +19,14 @@ sealed class Optional<out V> {
   object Absent : Optional<Nothing>()
 
   companion object {
+    @JvmStatic
+    fun <T> absent(): Optional<T> = Absent
+
+    @JvmStatic
+    fun <T> fromNullable(nullableReference: T?): Optional<T> {
+      return nullableReference?.let { Present(it) } ?: absent()
+    }
```

Comment:
This seems to duplicate `presentIfNotNull` below. Is this a matter of adding `@JvmStatic` on `presentIfNotNull` (not sure how doable this is without breaking ABI compatibility)